wsmaster/che-core-api-factory-bitbucket-server/src/main/java/org/eclipse/che/api/factory/server/bitbucket/server/AuthorizationSupplier.java

```diff
@@ -14,8 +14,11 @@
 import org.eclipse.che.api.factory.server.scm.exception.ScmCommunicationException;
 import org.eclipse.che.api.factory.server.scm.exception.ScmUnauthorizedException;
 
-/** Compute the Authorization header to sign the OAuth 1 request. */
-public interface AuthorizationHeaderSupplier {
+public interface AuthorizationSupplier {
```

Comment:
Can you please write javadoc to the interface? Is it bitbucket specific? Shouldn't it have Bitbucket in name then? Is it only oauth1 ?